docs/3_4_model_exchange_schema.adoc

```diff
@@ -18,16 +18,16 @@ The attributes in the following table are defined on top of the <<common-capabil
 |Attribute
 |Description
 
-|`modelIdentifier`
+|<<modelIdentifier>>
 |Short class name according to C syntax, for example, `A_B_C`.
 Used as prefix for FMI functions if the functions are provided in C source code or in static libraries, but not if the functions are provided by a DLL/SharedObject.
-`modelIdentifier` is also used as name of the static library or DLL/SharedObject.
+<<modelIdentifier>> is also used as name of the static library or DLL/SharedObject.
 See also <<header-files-and-naming-of-functions>>.
 
 |`completedIntegratorStepNotNeeded`
 |If `true`, function <<fmi3CompletedIntegratorStep>> need not be called (this gives a slightly more efficient integration).
```

Comment:
"(this gives a slightly more efficient integration)" : I do not understand this text. Integartion in the sens of solving the ODE? Could someone please improvethe eplanation. Should be non-normative anyway.

Reply:
I think it refers to the integration process. I also don't understand what the improvement in the performance is, but I rewrote it into non-normative text.

Reply:
more efficient because one call does not need to be made - that is written just before the comment.
Since this is confusing, I will remove this comment.